tools/ports/sdl2_image.py

```diff
@@ -5,8 +5,8 @@
 
 import os
 
-TAG = 'version_4'
-HASH = '30a7b04652239bccff3cb1fa7cd8ae602791b5f502a96df39585c13ebc4bb2b64ba1598c0d1f5382028d94e04a5ca02185ea06bf7f4b3520f6df4cc253f9dd24'
+TAG = 'release-2.6.0'
```

Comment:
It looks 2.6.3 is the newest release? https://github.com/libsdl-org/SDL_image/tags

Reply:
For whatever reason that is not under the `SDL2` branch, so I'm being conservative and going to the most recent release that lives under `SDL2` and also contains all the emscripten patches.

I don't claim to understand upstream's branching structure but I think this release is fine to use.

Reply:
Just to be clear,  `release-2.6.0` is the most recent tag that is part of the `SDL2` branch AFAICT.